settings.py

```diff
@@ -873,14 +873,14 @@
 
 # Setting the variable to True would mean that the primary index would become secondary and vice-versa
 # This should only be set to True after successfully running and verifying migration command on a particular index. 
-ES_APPS_INDEX_SWAPPED = True
-ES_CASE_SEARCH_INDEX_SWAPPED = True
-ES_CASES_INDEX_SWAPPED = True
-ES_DOMAINS_INDEX_SWAPPED = True
-ES_FORMS_INDEX_SWAPPED = True
-ES_GROUPS_INDEX_SWAPPED = True
-ES_SMS_INDEX_SWAPPED = True
-ES_USERS_INDEX_SWAPPED = True
+ES_APPS_INDEX_SWAPPED = False
+ES_CASE_SEARCH_INDEX_SWAPPED = False
+ES_CASES_INDEX_SWAPPED = False
+ES_DOMAINS_INDEX_SWAPPED = False
+ES_FORMS_INDEX_SWAPPED = False
+ES_GROUPS_INDEX_SWAPPED = False
+ES_SMS_INDEX_SWAPPED = False
+ES_USERS_INDEX_SWAPPED = False
```

Comment:
Publishing this change seems like it could be dangerous if self-hosters out there have overridden these settings and do not remove those overrides before you start the next reindex roll-out. This is where the safety feature outlined in [this doc](https://docs.google.com/document/d/10PLfGtEhOjdEtw3n2o7QEZhVaY_-X5IAVhaljv1hY_Y/edit#heading=h.y9lw4fpcbqut) would be good to have.

Reply:
I don't think it would affect them in any way as secondary index is set to None and `create_document_adapter` will ignore these setttings if secondary index is set to `None`

Reply:
Currently that is true, but could it become dangerous later when secondary index is changed to something other than `None` if the override in localsettings has not yet been removed?

Reply:
Yep. That is true. We will implement a failsafe before actually moving with ES 6 upgrade.